### PR TITLE
Update packages

### PR DESCRIPTION
**Pull request by sprynm**
Started with a broad package upgrade including to Gulp 4.  Required various trims and normalization to get the tests to pass on Windows.  Two small lint issues fixed.

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -16,10 +16,10 @@ gulp.task('lint', function() {
     .pipe(jshint.reporter( 'jshint-stylish' ))
 });
 
-gulp.task('test', ['lint'], function(done) {
-  return gulp.src('test/*.spec.js', {read: false})
+gulp.task('test', gulp.series('lint', function(done) {
+  return gulp.src('test/tags.spec.js', {read: false})
```
Comment (straker) [inline gulpfile.js]: Was there a particular reason you're only running `tags.spec` instead of all specs?

File: gulpfile.js [hunk 2]
```diff
@@ -61,9 +61,19 @@ gulp.task('sass', function() {
...
+    gulp.watch('assets/*.scss', gulp.series('sass'));
 });
 
-gulp.task('default', ['connect', 'sass', 'lint', 'test', 'watch']);
\ No newline at end of file
+gulp.task('default',
+    gulp.parallel('connect','sass', gulp.series( 'lint', 'test', 'watch'))
+);
+
+// var livingcss = require('gulp-livingcss');
```
Comment (straker) [inline gulpfile.js]: Are these comments still needed?

File: gulpfile.js (hunk 1 above)
Comment (sprynm) [reply gulpfile.js]: shoot, no. I had to quiet the noise during my bug hunt and this slipped in.

File: gulpfile.js (hunk 2 above)
Comment (sprynm) [reply gulpfile.js]: no. I was testing the plugin in the same space. Apologies, I will should have seen those in the merge.

**Comment by straker** [comment]
Awesome, looks good to me. Thanks again for doing all this.

**Pull request merged**